kast-json.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
python3 is not necesarily going to exist in your path when you have python 3 installed ,so this is not a good solution. It's better to have the script work with both python 2 or python 3, which is what it did before.